cnames_active.js

```diff
@@ -2322,6 +2322,7 @@ var cnames_active = {
   "quiz-app": "odevlord.github.io/Quiz-App", // noCF? (don´t add this in a new PR)
   "quizizz": "zaida04.github.io/Quizizz.js",
   "quizlet": "redyetidev.github.io/Quizlet.JS",
+  "quotes": "https://qts-xi.vercel.app",
```

Comment:
```suggestion
  "quotes": "cname.vercel-dns.com", // noCF
```
As the CI error notes, when using Vercel you should target this hostname rather than your app, and you should request the Cloudflare proxy be disabled with the `// noCF` comment.

Reply:
Ok, I committed new one